src/components/select/__internal__/select-list/select-list.style.ts

```diff
@@ -101,11 +100,17 @@ const StyledSelectListTableBody = styled.tbody<StyledSelectListProps>`
   height: ${({ listHeight }) => `${listHeight}px`};
 `;
 
-const StyledSelectListContainer = styled.div<
-  Pick<SelectListProps, "isLoading">
->`
+interface StyledSelectListContainerProps {
+  isLoading?: boolean;
+  placement?: string;
+}
+
+const StyledSelectListContainer = styled.div<StyledSelectListContainerProps>`
   background-color: white;
-  box-shadow: var(--boxShadow100);
+  box-shadow: ${({ placement }) =>
+    placement === "top"
+      ? "0 -5px 5px 0 #00141e33, 0 -10px 10px 0 #00141e1a"
```

Comment:
There is no design token for this so I just negated the y-offset of the original token value.

Reply:
Are there any design tokens that could be used instead of those hex colours?